### DISCUSSION TITLE
get remote rdp server info

### DISCUSSION THREAD

**Discussion by K3nshin66** [FreeRDP/FreeRDP#8444] [Q&A]
hi 
how can i get remote server name or domain (from remote server certificate) 
or list of recent users logged  when nla of remote computer is disabled 
this information its easy to get with windows rdp without credential
i use freeRDP-Sharp and i searched the setting and i did not find anything

i set these

	_settings->Authentication = 0;
        _settings->NlaSecurity = 0;

and connected was successfully but i don't know how can i retrieve these information

plz help....

**Comment by Saddingt0n** [comment]
Hi @sakabota. It's a bit off topic but I'm also using FreeRDP-Sharp and I can't seem to get it working. How do you get the connection to work? Could you share your contact details so I can message you privately?

**Reply by K3nshin66** [reply]
hi @Saddingt0n i add whole source file into my project (not installing from Nuget just downloaded from github) and modified "RDP.cs" 

	_settings->IgnoreCertificate = 1;
	_settings->Authentication = 0;
                _settings->NlaSecurity = 0;

**Reply by akallabeth** [reply]
FYI: https://github.com/FreeRDP/FreeRDP-Sharp is the place you are looking for, and the project is no longer maintained.

**Reply by K3nshin66** [reply]
yeap i add this one

**Reply by Saddingt0n** [reply]
> hi @Saddingt0n i add whole source file into my project (not installing from Nuget just downloaded from github) and modified "RDP.cs"
> 
> ```
> _settings->IgnoreCertificate = 1;
> _settings->Authentication = 0;
>             _settings->NlaSecurity = 0;
> ```

Hi @sakabota, very much appreciate your help. I tried that but it's still not working unfortunately.
It seems that the "settings" field in "RDP.cs" is not initialized so the program crashes with a NullReference exception.
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/91487625/205473157-e783454d-6f1d-4ee4-a2ff-80c4f629bdec.png)
Could you share your Telegram, email or what I can use to get in touch with you?
